clients/src/test/java/org/apache/kafka/clients/NetworkClientTest.java

```diff
@@ -198,6 +200,10 @@ public void testClose() {
         assertFalse(client.hasInFlightRequests(node.idString()));
         assertFalse(client.hasInFlightRequests());
         assertFalse(client.isReady(node, 0), "Connection should not be ready after close");
+        ApiVersions apiVersions  = TestUtils.fieldValue(client, NetworkClient.class, "apiVersions");
```

Comment:
nit: There is an extra space before `=` here and below.

I am not a huge fan of using `TestUtils.fieldValue`. Did you consider making both attributes package private or something like this instead? `ApiVersions` is passed to the constructor of `NetworkClient` so we could access it this way as well.